nwb_conversion_tools/datainterfaces/blackrockdatainterface.py

```diff
@@ -46,9 +47,9 @@ def get_metadata(self):
         comment = nsx_file.basic_header['Comment']
 
         metadata['NWBFile'] = dict(
```

Comment:
Does `pytz.timezone('EST').localize(session_start_time)` do something better than `.astimezone()` from `datetime`?

Reply:
localize will get a tz-naive datetime and add the tz info to that. Not sure if astimezone would do something else